taichi/backends/metal/aot_module_builder_impl.h

```diff
@@ -23,8 +24,9 @@ class AotModuleBuilderImpl : public AotModuleBuilder {
 
  private:
   const CompiledStructs *compiled_structs_;
+  BufferMetaData buffer_meta_data_;
   PrintStringTable strtab_;
-  std::vector<CompiledKernelData> kernels_;
+  TaichiAotData ti_aot_data;
```

Comment:
Private member names end with `_`

```suggestion
  TaichiAotData ti_aot_data_;
```